p2p/agent_test.go

```diff
@@ -48,7 +48,7 @@ func TestBroadcast(t *testing.T) {
 		}
 	}
 	u := func(_ context.Context, _ uint32, _ peerstore.PeerInfo, _ proto.Message) {}
-	bootnodePort := testutil.RandomPort()
+	bootnodePort := 14689
```

Comment:
Can we still random, but if we randomly get a port is used before, we randomly pick again?

Reply:
I think it's too much complexity,we have to save all the port opened